src/style.rs

```diff
@@ -266,6 +266,8 @@ pub enum Dimension {
     Points(f32),
     /// The dimension is stored in percentage relative to the parent item.
     Percent(f32),
+    /// The dimension is stored as the sum of percentage relative to the parent item and point.
+    PercentPoints(f32, f32),
```

Comment:
I would be more comfortable if this was a named; tuple types that contain the same internal value are a recipe for hard-to-spot bugs.